java/src/jmri/jmrix/xpa/XpaThrottle.java

```diff
@@ -38,6 +40,11 @@ public XpaThrottle(LocoAddress address, XpaTrafficController t) {
         log.debug("XpaThrottle constructor called for address {}", address);
     }
 
+    @Deprecated (since="5.13.3", forRemoval=true)
+    public XpaThrottle(LocoAddress address, XpaTrafficController t) {
+        this(address, t, jmri.InstanceManager.getDefault(jmri.jmrix.xpa.XpaSystemConnectionMemo.class));
+    }
```

Comment:
@bobjacobsen 
Is this deprecation needed? Or could the constructor be removed directly? I have hard to see that a script uses this constructor directly, or that some external program that's using JMRI as a library would use this constructor directly.